### PR TITLE
Replace --(target) with -t (target)

### PR DESCRIPTION
**Pull request by amirmc**
Related to https://github.com/mirage/mirage-www/issues/476

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,9 +65,9 @@ directly.
 
 ### To configure a unikernel before building:
 
-    $ mirage configure [config.ml] [--unix|--xen]
+    $ mirage configure [config.ml] [-t unix|-t xen]
```
Comment (amirmc) [inline README.md]: @yomimono Should this also be changed to `mirage configure [-f config.ml] [-t unix|-t xen]`?
i.e. adding the `-f` as per https://mirage.io/wiki/breaking-changes?

Comment (amirmc) [inline README.md]: Answered my own question by trying it :-P

**Comment by yomimono** [comment]
Sorry I missed this, @amirmc . There's a commit that does this already merged into the mirage-dev branch, and since we'll eventually merge that branch into master, I'd prefer to cherry-pick that commit in.

**Comment by yomimono** [comment]
Closing in favor of #166 .

**Pull request closed without merging**